src/lib/common/homestore_config.fbs

```diff
@@ -264,6 +264,9 @@ table Consensus {
 
     // Log difference to determine if the follower is in resync mode
     resync_log_idx_threshold: int64 = 100;
+
+    // Log difference from leader's point of view,  to determine if the follower is laggy.
```

Comment:
... to determine if the follower is laggy and if so, leader will stop pushing data until it drops under this threshold.